services/src/datasets/external/pangaea/meta.rs

```diff
@@ -68,6 +68,8 @@ impl PangeaMetaData {
             spatial_reference: SpatialReference::epsg_4326().into(),
             data_type: feature_type,
             columns: column_map,
+            time: None,
+            bbox: None,
```

Comment:
Same as for GFBio

Reply:
66619e218c2104c8577669937023a7354353f531